tests/cleo_tests/MemoryOperations/0AA4.txt

```diff
@@ -0,0 +1,24 @@
+{$CLEO .s}
+
+script_name "0AA4"
+debug_on
+
+trace "0AA4 get_dynamic_library_procedure"
+
+wait 0
+
+const LOAD_LIBRARY = 0x00858070
+int load_library_addr = read_memory LOAD_LIBRARY 4 false
```

Comment:
I usually divide test into multiple steps with logging, so if other opcode crashes it is possible to find it out in the log.